core/templates/dev/head/pages/exploration_editor/ParameterMetadataService.js

```diff
@@ -28,7 +28,7 @@ oppia.factory('parameterMetadataService', [
     var PARAM_SOURCE_ANSWER = 'answer';
     var PARAM_SOURCE_CONTENT = 'content';
     var PARAM_SOURCE_FEEDBACK = 'feedback';
-    var PARAM_SOURCE_PARAM_CHANGES = 'param_changes';
+    var PARAM_SOURCE_PARAM_CHANGES = 'paramChanges';
```

Comment:
I'm not entirely sure you should change this. It seems to be for a different purpose?

It may be fine, but make sure you understand what it's for and that changing it doesn't break anything.